### PR TITLE
refactor(case-stats): Daily case stats come from WHO ARCGIS endpoint

### LINKED ISSUE #1093: Daily case stats must come from WHO ARCGIS endpoint
Need to switch country daily stats from WHO's Sprinklr dashboard pipeline to WHO's ARCGIS pipeline.

Use of the ARCGIS endpoint approved by WHO.

Please switch to: https://services.arcgis.com/5T5nSi527N4F7luB/arcgis/rest/services/COVID19_hist_cases_adm0_v5_view/FeatureServer/0/query?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=ISO_2_CODE%2Cdate_epicrv%2CNewCase%2CCumCase%2CNewDeath%2CCumDeath%2C+ADM0_NAME&returnGeometry=true&featureEncoding=esriDefault&multipatchOption=xyFootprint&maxAllowableOffset=&geome

You need to update: https://github.com/WorldHealthOrganization/app/blob/master/server/appengine/src/main/java/who/RefreshCaseStatsServlet.java to store the aggregate data from ARCGIS in the existing data structure.

You should not change the existing schema of the data being stored.

### PR DESCRIPTION
**Pull request by myiremark**
Switch daily case stats from WHO's Sprinklr dashboard pipeline to WHO's ARCGIS pipeline.

Closes #1093 

#### Screenshots
N/A

#### Did you add any dependencies?
NO

#### How did you test the change?
* [ ] iOS Simulator
* [ ] Android Emulator
* [ ] iOS Device
* [ ] Android Device
* [x] `curl` to a dev App Engine server

From 'app/server' directory (2 terminals)

Terminal 1

git checkout master;
gradle clean && gradle build;
./bin/run-dev-server.sh;

Terminal 2

TMP_OUTPUT_DIR=~/Desktop/tmp/who-output/master;
mkdir -p $TMP_OUTPUT_DIR;

curl -XPOST http://localhost:8080/WhoService/getCaseStats -d '' -H "Content-Type: application/json" -H "Who-Client-ID:a67eda24-90c3-11ea-bb37-0242ac130002" -H "Who-Platform: WEB" | jq '.' > $TMP_OUTPUT_DIR/output-after-clean-build-getstats-before-cron.json

curl -i -H 'X-Appengine-Cron: true' -X GET http://localhost:8080/internal/cron/refreshCaseStats > $TMP_OUTPUT_DIR/output-internal-cron.txt;

curl -XPOST http://localhost:8080/WhoService/getCaseStats -d '' -H "Content-Type: application/json" -H "Who-Client-ID:a67eda24-90c3-11ea-bb37-0242ac130002" -H "Who-Platform: WEB" | jq '.' > $TMP_OUTPUT_DIR/output-after-clean-build-getstats-after-cron.json;

Terminal 1

(stop other process)

git checkout WHO_1093_arcgis_endpoint;
gradle clean && gradle build;
./bin/run-dev-server.sh;

Terminal 2

TMP_OUTPUT_DIR=~/Desktop/tmp/who-output/branch;
mkdir -p $TMP_OUTPUT_DIR;

curl -XPOST http://localhost:8080/WhoService/getCaseStats -d '' -H "Content-Type: application/json" -H "Who-Client-ID:a67eda24-90c3-11ea-bb37-0242ac130002" -H "Who-Platform: WEB" | jq '.' > $TMP_OUTPUT_DIR/output-after-clean-build-getstats-before-cron.json

curl -i -H 'X-Appengine-Cron: true' -X GET http://localhost:8080/internal/cron/refreshCaseStats > $TMP_OUTPUT_DIR/output-internal-cron.txt;

curl -XPOST http://localhost:8080/WhoService/getCaseStats -d '' -H "Content-Type: application/json" -H "Who-Client-ID:a67eda24-90c3-11ea-bb37-0242ac130002" -H "Who-Platform: WEB" | jq '.' > $TMP_OUTPUT_DIR/output-after-clean-build-getstats-after-cron.json;

diff -r ~/Desktop/tmp/who-output/master ~/Desktop/tmp/who-output/branch

Only meaningful change was this: not sure how it was getting calculated last time. its not returned in the updated response. it was in the original response.  Perhaps its configurable?

```
<     "lastUpdated": 1588965569998,
---
>     "lastUpdated": 1588896000000,
```

---

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).

- [x] REQUIRED: Do you have an Issue **assigned to you** for this PR?
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

**Comment by advayDev1** [comment]
Can you paste the output of before.json and after.json?  I suspect they are empty, unless you called the cron job in the test. @myiremark

**Comment by myiremark** [comment]
[who-output-before-and-after.zip](https://github.com/WorldHealthOrganization/app/files/4600263/who-output-before-and-after.zip)

Couldn't upload the JSON.

**Comment by myiremark** [comment]
I also turned down the cache from 10 minutes to 10 seconds temporarily.

**Comment by advayDev1** [comment]
@myiremark how did you test the change?
`curl -i  -H 'X-Appengine-Cron: true' -X GET http://localhost:8080/internal/cron/refreshCaseStats`

causes a 500 for me with this PR

**Comment by myiremark** [comment]
Good question.  I ran it last night and it worked fine.  It may have just been a cached response.  I did check the internal URL and had turned down the cron.yaml as well.  

I just switched the branch back and forth now and am getting the same error as you.

Let me check and get back to you.

**Comment by myiremark** [comment]
[tmp.zip](https://github.com/WorldHealthOrganization/app/files/4601238/tmp.zip)

**Comment by advayDev1** [comment]
Tested against production, only difference is timestamp.

File: .gitignore [hunk 1]
```diff
@@ -96,3 +96,10 @@ buck-out/
 
 # Bundle artifact
 *.jsbundle
+
+# Visual Studio
+
+.classpath
+.project
+.settings
+.vscode
```
Comment (advayDev1) [inline .gitignore:105]:
```suggestion
```

**Pull request merged**